api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -41,6 +41,26 @@ type VirtualMachineImageOSInfo struct {
 	Type string `json:"type,omitempty"`
 }
 
+// VirtualMachineImageInfo describes basic information related to the image, such as id and name. This information
```

Comment:
In future when VM Service supports other image formats, what would be the value of `id` here? If this is a unique identifier provided by the content provider, we should indicate that here.